cmake/Modules_CUDA_fix/upstream/FindCUDA.cmake

```diff
@@ -1451,10 +1431,10 @@ macro(CUDA_WRAP_SRCS cuda_target format generated_files)
           set(_cuda_fix_g3 TRUE)
         endif()
       endif()
-      set(_cuda_C_FLAGS "${CMAKE_${CUDA_C_OR_CXX}_FLAGS_${config_upper}}")
-      _filter_blacklisted_host_flags(_cuda_C_FLAGS)
```

Comment:
This was an important change that you lost.

Reply:
Ah. I thought this file was a mirror of the one in upstream. Just noticed this PR: https://gitlab.kitware.com/cmake/cmake/merge_requests/2167. Will bring back Tongzhou's changes.

Reply:
Thanks a lot. I really should finish that diff soon...